### PR TITLE
Add social links to some speakers

### PR DESCRIPTION
**Pull request by thewasif**
Adding social links to the profile of **Barend Scholtus**, **Bien Nguyen**, **Nam Phuong Nguyen Luong**, **Clif Kussmail**, **Eden Dang** and **Vu The Binh**
Here's the deployment link:
https://wasif.codes/vn.opentechsummit.asia/

### CODE REVIEW

**Comment by mariobehling** [comment]
Thanks. Please resolve conflicts.

**Comment by thewasif** [comment]
@mariobehling , conflicts have been resolved. Please check now

File: index.html [hunk 1]
```diff
@@ -490,8 +492,12 @@ <h1>Speakers</h1>
               <div class="image-holder">
                 <img alt="clif" src="img/ClifKussmail.jpg" width="300px">
                 <div class="hover-state text-center preserve3d">
-			<div class="social-links vertical-align">
-				<a href="https://www.facebook.com/ckussmaul"><i class="icon social_facebook"></i></a>
```
Comment (mariobehling) [inline index.html:494]: Already added

File: index.html [hunk 2]
```diff
@@ -457,7 +457,9 @@ <h1>Speakers</h1>
               <div class="image-holder">
                 <img alt="Eden" src="img/EdenDang.jpeg" width="300px">
                 <div class="hover-state text-center preserve3d">
-			<div class="social-links vertical-align">
```
Comment (mariobehling) [inline index.html:460]: Already added

File: index.html [hunk 3]
```diff
@@ -490,8 +492,12 @@ <h1>Speakers</h1>
               <div class="image-holder">
                 <img alt="clif" src="img/ClifKussmail.jpg" width="300px">
                 <div class="hover-state text-center preserve3d">
-			<div class="social-links vertical-align">
-				<a href="https://www.facebook.com/ckussmaul"><i class="icon social_facebook"></i></a>
+                        <div class="social-links vertical-align">
+                                <a href="https://www.linkedin.com/in/clifkussmaul/"><i class="icon fa fa-linkedin"></i></a>
+                                <a href="https://twitter.com/kussmaul"><i class="icon fa fa-twitter"></i></a>
+                                <a href="https://www.facebook.com/ckussmaul"><i class="icon fa fa-facebook"></i></a>
```
Comment (mariobehling) [inline index.html]: Linkedin and Twitter are already added

**Review by mariobehling — CHANGES_REQUESTED** [review]
Please go through your code. Only add people ones. Some links are there already.

**Comment by thewasif** [comment]
Fixed issues and added links for some more speakers
https://wasif.codes/vn.opentechsummit.asia/

**Review by mariobehling — APPROVED** [review]

**Pull request merged**